templates/initial-setup/connection_profile.php

```diff
@@ -84,21 +109,21 @@
             <div class="modal-content">
               <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-                <h4 class="modal-title" id="enterTokenModalModalLabel">Important</h4>
+                <h4 class="modal-title" id="enterTokenModalModalLabel"><?php _e( 'Important', 'wp-auth0' ) ?></h4>
               </div>
               <div class="modal-body">
                 <p>
-                  <?php _e( 'To complete the plugin\'s initial setup, you will need to enter your tenant Domain:', 'wp-auth0' ); ?>
+                  <?php _e( 'Enter your tenant Domain:', 'wp-auth0' ); ?>
                 </p>
-                <input type="text" name="domain" placeholder="youraccount.auth0.com" />
+                <input type="text" name="domain" class="js-a0-setup-input" placeholder="youraccount.auth0.com" required>
                 <br><br>
                 <p>
-                  <?php _e( 'And manually create an api token with the', 'wp-auth0' ); ?>
+                  <?php _e( 'Manually create an API token with the', 'wp-auth0' ); ?>
                   <a href="https://auth0.com/docs/api/management/v2/tokens#get-a-token-manually" target="_blank">
-	                  <?php echo __( 'token generator', 'wp-auth0' ); ?></a>
+	                  <?php _e( 'token generator', 'wp-auth0' ); ?></a>
```

Comment:
this won't last long (default expiration time). What do you need the token for?

Reply:
Setup process to create an Application and a DB connection